docs/develop/debug-office-add-ins-in-visual-studio.md

```diff
@@ -152,6 +170,21 @@ Next, Visual Studio does the following:
 
 ### Debug the Outlook add-in
 
+The best method for debugging an add-in in Visual Studio 2022 depends on whether the add-in is running in WebView2, which is the webview control that is associated with Microsoft Edge (Chromium), or an older webview control. To determine which webview control is being used, see [Browsers used by Office Add-ins](../concepts/browsers-used-by-office-web-add-ins.md). If your computer is using WebView2, see [Use the built-in Visual Studio debugger](#use-the-built-in-visual-studio-debugger). For any other webview control, see [Use the browser developer tools](#use-the-browser-developer-tools).
+
+#### Use the built-in Visual Studio debugger
+
+1. Set breakpoints, as needed, in the source JavaScript or TypeScript files. You can do this either before or after you start the add-in as described in the earlier section [Start the Excel, PowerPoint, or Word add-in project](#start-the-excel-powerpoint-or-word-add-in-project).
+
+1. When the add-in is running, use the add-in's UI to run the code that contains your breakpoints. 
+
+> [!TIP]
+>
+> - Sometimes in Outlook on the web, the Visual Studio debugger doesn't attach. If you get errors by the breakpoints that indicate they won't be hit, use the browser developer tools to attach to the Visual Studio debugger: After you have pressed F5 to start debugging and Outlook on the web has opened, follow the first four steps in the [Use the browser developer tools in Outlook on the web](#use-the-browser-developer-tools-in-outlook-on-the-web). (Use the instructions for Microsoft Edge (Chromium-based).) After you set a breakpoint in the browser tools and it is hit, execution will pause on the breakpoint in *both* the browser tools *and* in Visual Studio. This indicates that the Visual Studio debugger is attached. At this point, you can close the browser tools and add breakpoints in Visual Studio as you normally would.
+> - If you encounter any problems, there is more information at [Debug a JavaScript or TypeScript app in Visual Studio](../../../visualstudio/javascript/debug-nodejs.md).
```

Comment:
```suggestion
> - If you encounter any problems, there's more information at [Debug a JavaScript or TypeScript app in Visual Studio](/visualstudio/javascript/debug-nodejs?view=vs-2022&preserve-view=true).
```

Reply:
@Rick-Kirkham this link update should also address the build validation suggestion